base/sysimg.jl

```diff
@@ -79,6 +79,12 @@ end
 # vararg Symbol constructor
 Symbol(x...) = Symbol(string(x...))
 
+# Define the broadcast function, which is mostly implemented in
+# broadcast.jl, so that we can overload broadcast methods for
+# specific array types etc.
+#  --Here, just define fallback routines for broadcasting with no arguments
+broadcast(f) = f()
```

Comment:
Should `broadcast!` likewise first appear outside the `Broadcast` module? Ref. #18462, which this PR should close.